### ISSUE TITLE
Source code

### ISSUE THREAD

**Issue by itzme1on** [fudgeu/playlist#8]
Will you publish the source code of your mod? I would be happy to fork your repository and try to help you fix bugs, and I also have an idea with integration into Discord (yeah, I love working with discord so much :D)

**Comment by VinnyStalck** [comment]
Yes please, I would love to help with compatibility with mods that add music

**Comment by xlifi** [comment]
> Will you publish the source code of your mod? I would be happy to fork your repository and try to help you fix bugs, and I also have an idea with integration into Discord (yeah, I love working with discord so much :D)

Actually, you can decompile the code, modify it and even sell your modified version of the mod according to MIT license
